### PR TITLE
lodash > underscore

### PR DESCRIPTION
**Pull request by fyockm**
- update versions of modules in `package.json` and `templatePackage.json`
- fixed a few linting issues

### CODE REVIEW

File: lib/app/getPackageFile.js [hunk 1]
```diff
@@ -32,19 +32,19 @@ module.exports = function (config) {
...
+        'jade': '^1.3.1',
+        'moonboots-express': '^0.1.1',
+        'semi-static': '^0.0.5',
+        'serve-static': '^1.3.2'
     },
     hapi: {
-        "hapi": "^8.0.0",
-        "moonboots_hapi": "^4.0.0"
+        'hapi': '^8.0.0',
+        'moonboots_hapi': '^4.0.0'
```
Comment (lukekarrys) [inline lib/app/getPackageFile.js]: Should we update some of these if they are off by major versions? The ones I see that won't get the latest based on the `^` semver range are:

- helmet (0.7.1)
- moonboots-express (3.0.0)

File: lib/app/getPackageFile.js [hunk 2]
```diff
@@ -32,19 +32,19 @@ module.exports = function (config) {
...
-        "semi-static": "^0.0.5",
-        "serve-static": "^1.3.2"
+        'body-parser': '^1.4.3',
+        'compression': '^1.0.8',
+        'cookie-parser': '^1.3.2',
+        'express': '^4.6.1',
+        'helmet': '^0.3.2',
+        'jade': '^1.3.1',
+        'moonboots-express': '^0.1.1',
+        'semi-static': '^0.0.5',
```
Comment (lukekarrys) [inline lib/app/getPackageFile.js]: Should we update some of these if they are off by major versions? The ones I see that won't get the latest based on the `^` semver range are:

- helmet (0.7.1)
- moonboots-express (3.0.0)

Comment (fyockm) [inline lib/app/getPackageFile.js]: Totally missed those. Thanks for pointing them out!

**Comment by HenrikJoreteg** [comment]
+1 visually reviewed but not ran.

**Comment by fyockm** [comment]
@lukekarrys +1 now? would like to get this merged.

**Comment by lukekarrys** [comment]
+1!

**Comment by fyockm** [comment]
Thanks @lukekarrys!

**Comment by fyockm** [comment]
Version bump minor, patch? Thoughts?

**Comment by kamilogorek** [comment]
There has been no api changes nor added functionality so I opt for either patch or minor if we want to distinguish it slightly more.

**Comment by fyockm** [comment]
Looks like the other &-modules just did a patch bump, so followed suit with f908ee391fa0f707e8e00b05a75550a7780ceeca.

Looks like I don't have publish rights on `npm`. Can someone on the core team please publish? Or, add me as a maintainer? Thanks!

**Comment by wraithgar** [comment]
what's your npm username?

**Comment by fyockm** [comment]
same as github: fyockm

https://www.npmjs.com/~fyockm

**Comment by wraithgar** [comment]
Ok you're added, try now.

**Comment by fyockm** [comment]
Thanks @wraithgar! Published `ampersand@3.0.1`.

**Pull request merged**